src/Entity/ThematicCommunity/ThematicCommunityMembership.php

```diff
@@ -14,16 +18,31 @@
  * @ORM\DiscriminatorMap({
  *     "contact": "App\Entity\ThematicCommunity\ContactMembership",
  *     "adherent": "App\Entity\ThematicCommunity\AdherentMembership",
- *     "elected_representative": "App\Entity\ThematicCommunity\ElectedRepresentativeMembership",
  * })
  *
  * @Algolia\Index(autoIndex=false)
  */
 abstract class ThematicCommunityMembership
 {
-    public const MOTIVATION_PASSIVE = 'passive';
-    public const MOTIVATION_IDEA = 'idea';
-    public const MOTIVATION_MOBILISATION = 'mobilisation';
+    public const TYPE_ADHERENT = 'adherent';
+    public const TYPE_ELECTED_REPRESENTATIVE = 'electedRepresentative';
```

Comment:
should be removed

Reply:
no, it is still used for the admin filter